pylint/lint/pylinter.py

```diff
@@ -1491,7 +1493,7 @@ def _add_one_message(
                 message_definition.msgid,
                 message_definition.symbol,
                 MessageLocationTuple(
-                    abspath,
+                    abspath or "",
```

Comment:
It means we can raise a message without telling the user what file is affected, right ? I suppose there was a crash somewhere before if we passed `None` ?

Reply:
Yes!

There is probably a check somewhere for `if abspath`. That would fail both with `""` and `None`. However, when I made `MessageLocationTuple` I typed `abspath` and `module` to be `str`. In the commit that added `MessageLocationTuple` I also added the `or ""` to `module`. I think it makes sense to force `abspath` to be a string here.